version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "1.6.3-SNAPSHOT"
+version in ThisBuild := "1.7"
```

Comment:
- Removing 2.11 support should be a major version bump, not a minor (i.e. 2.0.0)
- New version number should still have 3 parts (major, minor, patch)
- Don't set the version to a non-SNAPSHOT version, because it's the release tag only that should have the plain version number (everything else is just a snapshot), and we create the release tag automatically.

Reply:
Thinking about it more, you might want to backdate this changelog entry to the last version that _actually_ supported 2.11